sdk/python/feast/repo_operations.py

```diff
@@ -112,6 +113,8 @@ def parse_repo(repo_root: Path) -> RepoContents:
                 res.data_sources.add(obj)
             if isinstance(obj, FeatureView):
                 res.feature_views.add(obj)
+                if isinstance(obj.stream_source, PushSource):
```

Comment:
maybe add a comment on how the batch source within a PushSource doesn't need to be defined on its own?

Reply:
I'll add this to the docs, in addition to the docstring fro feature views.